skills/studio/scripts/studio/utils/eval_judge.py

```
@@ -0,0 +1,328 @@
+        """Advisory PASS/FAIL/UNKNOWN on rule-compliance; never contributes to the exit code."""
+        if run is None:
+            return self._result(VERDICT_UNKNOWN, ["run artifacts could not be loaded"],
+                                "unscoreable: no readable run", scenario)
+        if self._judge_fn is None:
+            return self._result(VERDICT_UNKNOWN, ["no judge model wired (advisory)"],
+                                "no judge_fn: model supplied out-of-tree", scenario)
+        request = build_judge_request(run, scenario)
+        try:
+            reply = self._judge_fn(request)
+        # A misbehaving injected judge must degrade to UNKNOWN, never sink the run.
+        except Exception as exc:  # pylint: disable=broad-except
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)